### PR TITLE
GitHub Actions: don't decompress kernel on ARM arch

### PR DESCRIPTION
**Pull request by uncleDecart**

### CODE REVIEW

File: .github/workflows/assets.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@ jobs:
           # hand-off part between the two. Therefore for now unpack the kernel.
           if [ "${{ env.ARCH }}" = arm64 ]; then
              mv assets/kernel assets/kernel.gz
```
Comment (rene) [inline .github/workflows/assets.yml:81]: The kernel is not compressed, I think this entire run can be removed...

Comment (eriknordmark) [reply .github/workflows/assets.yml:81]: Then the whole code block including the FIXME comment can be dropped. Maybe leaving a comment saying that we assume the kernel is not compressed?

Comment (uncleDecart) [reply .github/workflows/assets.yml:81]: well we don't decompress amd64 kernel, so I think it's safe just to drop this step without any comment. Now it's default behaviour

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**